scripts/postcopy.js

```diff
@@ -0,0 +1,24 @@
+
+module.exports = async function(fs, path, log, options, done) {
+  const buildPath = options.outputdir;
+  const coursePath = `${buildPath}course`;
+  const data = await fs.promises.readFile(`${coursePath}/config.json`).catch(error => {
+    log(error);
+  });
+  if (!data) return done();
+  const config = JSON.parse(data.toString());
+  const scormVersion = config?._spoor?._advancedSettings?._scormVersion ?? '1.2';
+  const scormPath = `${options.plugindir}/scorm/${scormVersion}`;
+  const files = await fs.promises.readdir(`${scormPath}`).catch(error => {
+    log(error);
+  });
+  if (!files) return done();
+  //log(`Adding SCORM ${scormVersion} files`);
```

Comment:
Minor: commented code needs removed

Reply:
I had left that in on purpose in case we did want to output it to the console when compiling, but decided we wouldn't include as default.